docs/Lua API.rst

```diff
@@ -4305,41 +4305,44 @@ xlsxreader
 ==========
 
 Utility functions to facilitate reading .xlsx spreadsheets. It provides the
-following API methods:
+following low-level API methods:
 
  - ``file_handle open_xlsx_file(filename)``
  - ``close_xlsx_file(file_handle)``
  - ``sheet_names list_sheets(file_handle)``
  - ``sheet_handle open_sheet(file_handle, sheet_name)``
  - ``close_sheet(sheet_handle)``
- - ``cell_strings get_row(sheet_handle)``
+ - ``cell_strings get_row(sheet_handle, max_tokens)`` The ``max_tokens``
+   parameter is optional. If set to a number > 0, it limits the number of cells
+   read and returned for the row.
 
- Example::
+ Lua users would be better off using the Lua class wrappers, though. For
+ example::
```

Comment:
It would be nice to have docs for the wrappers long-term, although I think the example is good and that's fine with me for now. This phrasing sounds to me like the classes were mentioned already - were they? If not, it would be worth at least calling out that the `XlsxReader` class (or whatever the recommended method for creating one is) exists before we get into the example. Perhaps mentioning that `XlsxSheetReader` exists in advance would help too, to minimize confusion with `XlsxReader`.

Reply:
good point. docs added.